### ISSUE TITLE
mute file duplicated during training

### ISSUE THREAD

**Issue by SayanoAI** [fumiama/Retrieval-based-Voice-Conversion-WebUI#8]
what's the rationale for duplicating the mute files during training?

![image](https://github.com/fumiama/Retrieval-based-Voice-Conversion-WebUI/assets/28544690/d0949d00-5366-4e88-abce-593fc1a65a2f)

Why does including it just once not work?

**Comment by blaise-tk** [comment]
`if_f0_3` is always true, because you can't extract without a f0 method, idk why that is there

**Comment by 413x1nkp** [comment]
> `if_f0_3` is always true, because you can't extract without a f0 method, idk why that is there

(unrelated to OP's issue) honestly i'm extremely tired of all the weird names for vars and functions that are in RVC.
i'll start working on renaming all of the vars to be something that is easier to understand

**Comment by fumiama** [comment]
> i'll start working on renaming all of the vars to be something that is easier to understand

Thanks but maybe you can start it later to aviod code conflicts because I'm changing the whole code structure recently.

As for the main topic of this issue, I will remove all unnecessary vars in train later. Now I am just working on the infer part.

**Comment by SayanoAI** [comment]
> Thanks but maybe you can start it later to aviod code conflicts because I'm changing the whole code structure recently.
> 
> As for the main topic of this issue, I will remove all unnecessary vars in train later. Now I am just working on the infer part.

I'd recommend structuring the RVC code as a pypi package and build the gui on top of that. This will make it easier for people to create colabs and whatnot.

**Comment by fumiama** [comment]
> I'd recommend structuring the RVC code as a pypi package

Yes. See the progress at https://github.com/RVC-Project/Retrieval-based-Voice-Conversion .

But now, I'm just letting the RVC code seeing more like a package so that we can migrate to package version easily.
